clean_data.py

```diff
@@ -188,21 +179,25 @@ def main(argv):
       for line in fp:
         line = line.strip()
         print(create_hmac(line, FLAGS.hmac_key, hash_type=hashlib.sha3_384))
-  return
+    return
 
   df = pd.read_csv(FLAGS.input,
                    on_bad_lines='warn',
-                   na_values=['NR', '\n',
+                   na_values=['', '\n',
```

Comment:
You should add a comment, here, or in the function definition, stating what the policy is.